### ISSUE TITLE
Avoid scary "process crashed" message when inadequate.

### ISSUE THREAD

**Issue by ceraolo** [OpenModelica/OpenModelica#11174]
### Description
When the model does not find the imput file of a combi table, simulation must stop, but the message "process creassed" is inadequate.

### Steps to Reproduce
Consider the following model:  
```
model NoTableName
  Modelica.Blocks.Sources.CombiTimeTable noTableName(tableOnFile = true, 
      fileName = "Test.txt");
equation
end NoTableName;

```
If we run it, we get the following screenshot:
![image](https://github.com/OpenModelica/OpenModelica/assets/7373030/a3cd43da-9f75-47df-bf22-0d068d16c379)

I thing that the message "process crashed" is really to scary for tthis case.
Something similar occurs with the following model 
```
model WithTableName
  Modelica.Blocks.Sources.CombiTimeTable withTableName(fileName = "Test.txt", 
     tableOnFile = true, tableName = "MyName");
equation
end WithTableName;
```

![image](https://github.com/OpenModelica/OpenModelica/assets/7373030/62553ce4-0484-48d2-93ee-1e904c5a9da2)

### Expected Behavior
in the screenshots above I would expect not to read the two rows containing the text "process crashed".

### Version and OS
OM 1.22.0-dev301 (win64)

**Comment by casella** [comment]
I agree. However, we have a lot of higher priority issues to fix first.

**Comment by ceraolo** [comment]
agree. 
I've noticed this several years ago; I waited till now exactly because it was not very important.
I chose this moment since OM's general MMI has imprroved alot, so smaller issues are more evident now

**Comment by bilderbuchi** [comment]
Maybe related to memory leaks in CombiTimeTable? https://github.com/OpenModelica/OpenModelica/issues/5775 Could that make the process crash instead of terminate in an orderly fahsion in case the input file is not found?
